stdlib/list.gr

```diff
@@ -742,6 +826,11 @@ provide let rec takeWhile = (fn, list) => {
  * @param list: The list to search
  * @returns `Some(element)` containing the first value found or `None` otherwise
  *
+ * @example
+ * List.find(a => a < 7 && a > 4, [1, 2, 3, 8, 9, 5, 1, 8]) == Some(5)
```

Comment:
This might be better off using chars. just so its substantially different then `findIndex` it just seems like needless confusion not that the example doesnt work chars would just be more clear. maybe something like: 

```
List.filter(a => a == 'a',  [ 'b', 'b', 'a', 'd', 'a']) == Some(2)
```

The simpler example also makes it more clear while the examples should not be the number one source of knowledge and if you read the description it is clear what it does the more info you can give with an example the better.